synapse/lib/cell.py

```diff
@@ -4494,13 +4617,13 @@ async def getCellInfo(self):
                     'https': self.https_listeners,
                 }
             },
-            'features': {
-                'tellready': True,
-                'dynmirror': True,
-            },
+            'features': self.features,
```

Comment:
do we care that this changes the features results from bool to ints? I def want it for the telepath stuff, but we could always generate a dict of bools dynamically if we need to...